chain/consensus/mir/README.md

```diff
@@ -0,0 +1,42 @@
+# Mir Consensus
+
+The code in this package integrates the Mir framework with the implementation over Mir
+of a pBFT consensus. This consensus can be used as an alternative consensus to FilecoinEC
```

Comment:
why is **p**BFT?

Reply:
practical BFT: https://crushcrypto.com/what-is-practical-byzantine-fault-tolerance/